src/Application/RedisMatchingStorage.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace MatchBot\Application;
+
+class RedisMatchingStorage implements RealTimeMatchingStorage
+{
+    public function __construct(private \Redis $redis) {}
```

Comment:
all functions in this class should just be delegating to redis so that this acts as a transparent wrapper around redis and does not change the behaviour in any way.